packages/@sanity/cli/src/cli.js

```diff
@@ -69,6 +69,22 @@ module.exports = async function runCli(cliRoot) {
 }
 
 function getCoreModulePath(workDir) {
+  /* eslint-disable no-process-env */
+  if (process.env.SANITY_CLI_CORE_PATH) {
+    const local = fse.existsSync(process.env.SANITY_CLI_CORE_PATH)
+    if (local) {
+      console.log(
+        chalk.yellow(`[WARN] Loading @sanity/core from: ${process.env.SANITY_CLI_CORE_PATH}`)
+      )
+      return process.env.SANITY_CLI_CORE_PATH
+    }
+    console.warn(
+      chalk.red(`[ERR] Could not resolve SANITY_CLI_CORE_PATH: ${process.env.SANITY_CLI_CORE_PATH}`)
```

Comment:
Could we make the error message clearer / make it more "human" friendly and aligned with the rest of the messages in the cli (for example, removing the `[ERR]`)? 🙏

Reply:
👍  We have mixed uses of `[ERR]` in this file. Happy to go with whichever aligns with the rest of the CLI.